web/book/src/project/contributing/development.md

```diff
@@ -435,37 +437,44 @@ task run-playground
 
 Currently we release in a semi-automated way:
 
-1. PR & merge an updated
-   [Changelog](https://github.com/PRQL/prql/blob/main/CHANGELOG.md). GitHub will
-   produce a draft version at <https://github.com/PRQL/prql/releases/new>,
-   including "New Contributors".
+- PR & merge an updated
```

Comment:
Why the change from numbered? I obv don't have a super-strong view, but I had thought that making them numbered showed they were ordered?

Reply:
Good point. Updated.